### PR TITLE
Hyundai: Car Port for AZERA Hybrid 6TH Generation

### PR DESCRIPTION
**Pull request by haram-KONA**
Hyundai Azera Hybrid 2020-2022 Car Support 

Dongle ID : 66189dd8ec7b50e6

test route 

"66189dd8ec7b50e6|2023-09-19--15-12-51"

### CODE REVIEW

**Comment by sshane** [comment]
Can you put the device on wifi so it can upload rlogs?

**Comment by haram-KONA** [comment]
If i try to turn off the car after connecting the comma power and driving, the car will not turn off. 
If i remove the comma power again in that state, the engine will turn off normally. 

What is the reason for this?  
OMG ~~~~

File: selfdrive/car/tests/routes.py [hunk 1]
```diff
@@ -94,6 +94,7 @@ class CarTestRoute(NamedTuple):
   CarTestRoute("f44aa96ace22f34a|2021-12-22--06-22-31", HONDA.CIVIC_2022),
 
   CarTestRoute("87d7f06ade479c2e|2023-09-11--23-30-11", HYUNDAI.AZERA_6TH_GEN),
+  CarTestRoute("66189dd8ec7b50e6|2023-09-19--15-12-51", HYUNDAI.AZERA_HEV_6TH_GEN, segment=4),
```
Comment (sshane) [inline selfdrive/car/tests/routes.py:97]:
```suggestion
  CarTestRoute("66189dd8ec7b50e6|2023-09-20--07-02-12", HYUNDAI.AZERA_HEV_6TH_GEN, segment=4),
```

Comment (sshane) [reply selfdrive/car/tests/routes.py:97]: @haram-KONA which one is okay to be public?

**Comment by sshane** [comment]
@chanhojung can you send a route where it stays on?

**Comment by sshane** [comment]
I checked some routes where your lights turned off before ignition and it shows that the bus is still active the entire time with no CAN errors. Even the ignition signal on CAN remains 1. How are you turning your car off exactly? Can you describe the steps?

**Comment by haram-KONA** [comment]
> I checked some routes where your lights turned off before ignition and it shows that the bus is still active the entire time with no CAN errors. Even the ignition signal on CAN remains 1. How are you turning your car off exactly? Can you describe the steps?


@chanhojung 
There's nothing special about turning off my car. Stop the vehicle, apply the brakes, and press and hold the start button to turn off the engine. At this time, the EPB is automatically engaged.

**Comment by sshane** [comment]
Strange indeed, one ignition switch signal shows some change, but another doesn't. Can you take a photo of your instrument cluster before pressing power button and after?

image

**Pull request closed without merging**